serverless.yml

```diff
@@ -218,6 +232,7 @@ custom:
     SIGNING_DID_KEY: ${env:SIGNING_DID_KEY, ""}
     SIGNING_DID_PRIVATE_KEY: ${env:SIGNING_DID_PRIVATE_KEY, ""}
     REVOCATION_OCSP: ${env:REVOCATION_OCSP, ""}
+    REVOCATION_OCSP_API_KEY: ${env:REVOCATION_OCSP_API_KEY, ""}
```

Comment:
Will need to add into AWS prod env before merging

Reply:
Added in prd